src/python/qeopenfermion/_utils.py

```diff
@@ -207,3 +209,74 @@ def evaluate_qubit_operator(qubit_operator: QubitOperator,
 	value_estimate = ValueEstimate(total)
 	return value_estimate
 
+
+def reverse_qubit_order(qubit_operator, n_qubits=None):
+    """Reverse the order of qubit indices in a qubit operator.
+
+    Args:
+        qubit_operator (zmachine.core.qubitoperator.QubitOperator): the operator
```

Comment:
it's not from `zmachine` ;)

Reply:
Also, I think now we actually use `QubitOperator` directly from openfermion.

Reply:
Addressed here:
be27da9